metrics/report/report_dockerfile/scaling.R

```diff
@@ -126,9 +186,11 @@ mem_stats_plot = suppressWarnings(ggtexttable(data.frame(rstats),
 	))
 
 # plot how samples varied over  'time'
-mem_line_plot <- ggplot() +
-	geom_line( data=data, aes(count, avail_gb, colour=testname, group=dataset), alpha=0.2) +
-	geom_smooth( data=data, aes(count, avail_gb, colour=testname, group=dataset), se=FALSE, method="loess", size=0.3) +
+mem_line_plot <- ggplot(data=fndata, aes(as.numeric(as.character(pod)), 
```

Comment:
I see you pass the data at the ggplot() level, not to each graph - which is fine, and the normal ggplot idiom - but, iirc, there was some reason in the past I didn't do that - so, we'll just have to keep our eyes on that ;-)